unified-runtime/source/adapters/cuda/usm.cpp

```diff
@@ -363,9 +360,6 @@ UR_APIEXPORT ur_result_t UR_APICALL urUSMPoolCreate(
   }
   return UR_RESULT_SUCCESS;
 #else
-  std::ignore = Context;
-  std::ignore = PoolDesc;
-  std::ignore = Pool;
```

Comment:
Should we `(void)` the ones in this file so we don't get the warnings in debug mode?

Reply:
Good catch. I'm happy either (void)ing them in the body, or annotating the parameter with `[[maybe_unused]]`. Perhaps the latter is better for keeping the body clear?

Reply:
Yeah `[[maybe_unused]]` sounds good

Reply:
done

Reply:
The ones here `Context`, `PoolDesc` and `Pool` still aren't marked `[[maybe_unused]]`